src/tool_help.c

```diff
@@ -467,8 +468,8 @@ static const struct helptxt helptext[] = {
    "Transfer based on a time condition"},
   {"    --tls-max <VERSION>",
    "Set maximum allowed TLS version"},
-  {"    --tls13-ciphers <list>",
-   "TLS 1.3 ciphersuites (OpenSSL)"},
+  {"    --tls13-ciphers <list of TLS 1.3 ciphersuites>",
```

Comment:
This seems pretty verbose and will probably go over the 79 column limit, what about just `<TLS 1.3 ciphersuites>` or `<ciphers>`?

Reply:
It doesn't go over the limit... `gen.pl` warns if it does.